Drivers/SiteSearchDisplay.cs

```diff
@@ -70,7 +70,7 @@ public override async Task<IDisplayResult> DisplayAsync(SiteSearch part, BuildPa
 
         public override async Task<IDisplayResult> EditAsync(SiteSearch part, BuildPartEditorContext context)
         {
-            if (part.ContentTypeSettings == null)
+            if (part.ContentTypeSettings == null || part.ContentTypeSettings.Count() == 0)
```

Comment:
Instead of `part.ContentTypeSettings.Count() == 0`, use `!part.ContentTypeSettings.Any()`